srcpkgs/dpdk/template

```diff
@@ -0,0 +1,87 @@
+# Template file for 'dpdk'
+pkgname=dpdk
+version=19.08
+revision=1
+archs="x86_64 i686"
+build_style=meta
+hostmakedepends="cmake meson doxygen python3-Sphinx pkg-config elfutils"
+makedepends="libnuma-devel linux-headers jansson-devel libpcap-devel zlib-devel libressl-devel libbsd-devel libelf libmnl"
+depends="jansson zlib libbsd libressl python-pyelftools"
+short_desc="Data Plane Development Kit"
+maintainer="Hans-J. Schmid <knock@myopendoor.de>"
+license="BSD-3-Clause, GPL-2.0-only"
+homepage="https://www.dpdk.org/"
+distfiles="https://github.com/DPDK/${pkgname}/archive/v${version}.tar.gz"
+checksum=1ceff1a6f4f8d5f6f62c1682097249227ac5225ccd9638e0af09f5411c681038
+
+do_configure() {
+	meson --prefix /usr -Denable_kmods=false -Denable_docs=true build
+}
+
+do_build() {
+	ninja -C build
+}
+
+do_install() {
+	DESTDIR=${DESTDIR} ninja -C build install
+	vmkdir usr/lib/modules-load.d
+}
+
+post_install() {
+	vlicense license/bsd-3-clause.txt
+	vlicense license/gpl-2.0.txt
+}
+
+dpdk-doc_package() {
+	short_desc+=" - documentation"
+	archs=noarch
+	pkg_install() {
+		vmove usr/share/doc
+	}
+}
+
+dpdk-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision} jansson-devel zlib-devel libbsd-devel libressl-devel"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/bin/dpdk-pdump
+		vmove usr/bin/dpdk-proc-info
+		vmove usr/bin/dpdk-test*
+		vmove usr/include
+		vmove usr/lib/*.so
+		vmove usr/lib/*.a
+		vmove usr/lib/dpdk/pmds-${version}/*.so
+		vmove usr/lib/pkgconfig
+		vmove usr/share/dpdk/examples
+	}
+}
+
+dpdk-libs_package() {
+	short_desc+=" - runtime libraries"
+	pkg_install() {
+		vmove usr/lib/*.so.*
+		vmove usr/lib/dpdk/pmds-${version}/*.so.*
+	}
+}
+
+dpdk-dkms_package() {
+	short_desc+=" - DKMS kernel modules"
+	dkms_modules="dpdk-igb-uio $version dpdk-rte-kni $version"
+	depends="dkms dpdk-devel"
```

Comment:
It is a dkms modules, so I doubt that it needs `dpdk-devel` to compile and `-devel` packages should normally not be runtime requirement of normal packages (ofc exceptions can occur)

Reply:
It needs the devel package.

Reply:
These are the includes from the devel package:
#include <rte_pci_dev_features.h>
#include <rte_kni_common.h>